bmaptools/BmapCreate.py

```diff
@@ -155,8 +155,9 @@ def __init__(self, image, bmap, chksum_type="sha256"):
         try:
             self._cs_len = len(hashlib.new(self._cs_type).hexdigest())
         except ValueError as err:
-            raise Error("cannot initialize hash function \"%s\": %s" %
-                        (self._cs_type, err))
+            raise Error(
+                'cannot initialize hash function "%s": %s' % (self._cs_type, err)
+            )
```

Comment:
This sort of change is controversial, but ... I do not like it personally, but I also think current code needs lots of clean-ups. So anyway, overall it is better to follow "black" suggestions. Thanks!

Reply:
I agree in not liking some of these formatting changes, but that it makes sense for now, for gaining consistency and adhering to a certain code style for further clean-ups.